src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder.java

```diff
@@ -150,8 +149,8 @@ public boolean isVerifyServices() {
   }
 
   @VisibleForTesting
-  void setAfterBuildStep(KubeConfigAfterBuildStep afterBuildStep) {
-    this.afterBuildStep = afterBuildStep;
+  void setBuildStep(KubeConfigBuildStep buildStep) {
```

Comment:
This method and instance variable should retain the name afterBuildStep. The intent behind this method is to enable after build cleanup during testing.